controllers/git_tekton_resources_renovater_test.go

```diff
@@ -166,5 +152,22 @@ var _ = Describe("Git tekton resources renovater", func() {
 			Expect(allEvents[0].Reason).To(Equal("ErrorReadingPaCSecret"))
 			deleteComponent(componentNamespacedName)
 		})
+
+		It("It should not trigger job", func() {
+			installedRepositoryUrls := []string{
+				"https://github/test/repo1",
+				"https://github/test/repo2",
+			}
+			github.GetAppInstallations = func(appIdStr string, privateKeyPem []byte) ([]github.ApplicationInstallation, string, error) {
+				repositories := generateRepositories(installedRepositoryUrls)
+				return []github.ApplicationInstallation{generateInstallation(repositories)}, "slug", nil
+			}
+			componentNamespacedName := createComponentForPaCBuild(getComponentData(componentConfig{componentKey: types.NamespacedName{Name: "testnottrigger"}, gitURL: "https://github/test/repo3"}))
+			createDefaultBuildPipelineRunSelector(defaultSelectorKey)
+			// try to wait for renovate Reconcile at least start
+			time.Sleep(2 * time.Second)
```

Comment:
What does it mean by "at least start"?

For what to wait for renovate Reconcile as least start?

Reply:
well JUST in case it would really create some jobs ;D

Reply:
I don't think it's needed.

Reply:
I mean without any wait for renovate, this test doesn't test anything

so with this test makes at least some sense, and it isn't such huge delay